src/components/componentGroup.ts

```diff
@@ -23,6 +23,48 @@ export module Component {
       components.forEach((c: AbstractComponent) => this._addComponent(c));
     }
 
+    /**
+     * Retrieves closest PlotData to queryPoint across all plots in this group.
+     * Plots define their own notion of closeness; this function compares results across
+     * plots and picks the closest to queryPoint using the Euclidean norm. Ties are
+     * broken by favoring the plots higher in the group.
+     *
+     * @param {Point} queryPoint The point to which plot data should be compared
+     *
+     * @returns {PlotData} The PlotData closest to queryPoint
+     */
+    public getClosestPlotData(queryPoint: Point): Plot.PlotData {
+      var minDistSquared = Infinity;
+      var closestPlotData: Plot.PlotData = {
+        data: [],
+        pixelPoints: [],
+        plot: null,
+        selection: d3.selectAll([])
+      };
+
+      this.components().forEach((c) => {
+        // consider only components implementing this function
+        if ((<any>c).getClosestPlotData !== undefined) {
```

Comment:
We should not be anycasting this.  You should be able to do a call like...

if (c instanceof Plot.AbstractPlot)

I think